autodoc-admin-api/data.lua

```diff
@@ -189,7 +191,78 @@ return {
           ]],
         },
       }
-    }
+    },
+    config = {
+      title = [[Db-less configuration]],
+      description = "",
+      ["/"] = {
+        POST = {
+          title = [[Update db-less config]],
+          endpoint = [[<div class="endpoint post">/config</div>]],
+          description = [[
+            This endpoint allows resetting a db-less Kong with a new
+            yml data file.
+
+            To learn more about the yaml file needed, please run:
+
+            ```
+            kong config init
+            ```
+
+            That will generate a file with the appropiate structure, names
+            and examples.
+          ]],
+          response = [[
+            ```
+            HTTP 200 OK
+            ```
+
+            ``` json
+            {
+                { "services" = [],
+                  "routes" = []
+                }
+            }
+            ```
+
+            The response contains a list of all the entities that were parsed from the
+            input file.
+          ]]
+        }
+      },
+    },
+    tags = {
+      title = [[ Tags Admin list ]],
```

Comment:
I think the title is unclear (I wondered for a second what an "admin list" is). Perhaps just "Tags" would avoid the confusion?